commands/timeout.js

```diff
@@ -24,58 +36,52 @@ module.exports = {
       return;
     }
 
-    for (var [id, user] of message.mentions.users) {
```

Comment:
We're removing support for multiple users at once? I think it's cleaner but might still be the expected behavior for some staff?